### PR TITLE
Initial documentation for https app

### PR DESCRIPTION
**Pull request by hkortti**
Initial user documentation for the https app.

### CODE REVIEW

File: docs/apps/https.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Example using wget:
+```
+wget https://http-app.badrap.io/app/claim/1abc...
+```
+
+Example using curl:
+```
+curl https://http-app.badrap.io/app/claim/1abc...
+```
+
```
Comment (jviide) [inline docs/apps/https.md:42]: Currently the wget and curl example paragraphs are not indented to the same level as the rest of the step:

Screenshot 2021-05-14 at 11 45 05

Could this be fixed?

**Review by jviide — CHANGES_REQUESTED** [review]
The text looks good 👍 There are some changes coming to the HTTPS app UI soon, let's wait for that to happen and update the screenshots accordingly.

File: docs/apps/https.md (hunk 1 above)
Comment (hkortti) [reply docs/apps/https.md:47]: Fixed

**Comment by hkortti** [comment]
Screenshots updated and text changed to reflect newest UI changes.

**Review by jviide — APPROVED** [review]
LGTM 👍

**Pull request merged**